CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- CocoaPods podspecs removed. Blueprint will only be vended via Swift Package Manager.
```

Comment:
Do you know if there's anything we could/should do with on cocoapods trunk to signal that these pods are inactive/archived? Probably not critical, just curious if there's some final cleanup we should be doing here.

Reply:
I don't think so.